2gp-demo/force-app/main/default/permissionsets/Feature_Access_Demo_User.permissionset-meta.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<PermissionSet xmlns="http://soap.sforce.com/2006/04/metadata">
+    <classAccesses>
+        <apexClass>FeatureAccessDemoController</apexClass>
+        <enabled>true</enabled>
+    </classAccesses>
+    <description>Provides access to pages, classes, and tabs required by the PLP &quot;Feature Access&quot; demo.</description>
+    <hasActivationRequired>false</hasActivationRequired>
+    <label>Feature Access Demo User</label>
```

Comment:
please rename to "Feature Access Demo", and remove "User" from filename as well. This perm set is referenced with the shorter name in doc

Reply:
OK, will do.